tldw_Server_API/tests/LLM_Adapters/integration/test_adapters_chat_endpoint.py

```
@@ -27,61 +43,94 @@ def _payload(stream: bool = False):
+    real = _real_key("openai")
+    if real:
+        chat_endpoint.API_KEYS = {**(chat_endpoint.API_KEYS or {}), "openai": real}
+        from tldw_Server_API.tests._plugins.chat_fixtures import get_auth_headers
+        headers = get_auth_headers(auth_token, getattr(client, "csrf_token", ""))
+        with client.stream("POST", "/api/v1/chat/completions", json=_payload(stream=True), headers=headers) as resp:
+            assert resp.status_code == 200
+            ct = resp.headers.get("content-type", "").lower()
+            assert ct.startswith("text/event-stream")
+            lines = list(resp.iter_lines())
+            # Should include chunks and single DONE
+            assert any(l.startswith("data: ") and "[DONE]" not in l for l in lines)
```

Ruff (0.14.4)

Ambiguous variable name: `l`

(E741)